airflow/api/common/experimental/mark_tasks.py

```diff
@@ -191,14 +191,16 @@ def set_state(task, execution_date, upstream=False, downstream=False,
     return tis_altered
 
 
-def set_dag_run_state(dag, execution_date, state=State.SUCCESS, commit=False):
+def set_dag_run_state(dag, execution_date, state=State.SUCCESS, commit=False,
+                      include_task_instance=True):
     """
     Set the state of a dag run and all task instances associated with the dag
     run for a specific execution date.
     :param dag: the DAG of which to alter state
     :param execution_date: the execution date from which to start looking
     :param state: the state to which the DAG need to be set
     :param commit: commit DAG and tasks to be altered to the database
+    :param include_task_instance: set state of task instance if set to true
```

Comment:
s/task_instance/task_instances (here and elsewhere), e.g. "set state of task instances in the dag run"